shared/teams/channel/header.tsx

```diff
@@ -0,0 +1,198 @@
+  )
+
+  const bottomDescriptorsAndButtons = (
+    <Kb.Box2 direction="vertical" alignSelf="flex-start" gap="tiny">
+      {!!description && (
+        <Kb.Text type="Body" lineClamp={3}>
+          {description}
+        </Kb.Text>
+      )}
+      {numParticipants !== -1 && (
+        <Kb.Text type="BodySmall">
+          {numParticipants.toLocaleString()} {pluralize('member', numParticipants)}
+          {!!newMemberCount && ' · ' + newMemberCount.toLocaleString() + ' new this week'}
+        </Kb.Text>
+      )}
+      <Kb.Box2 direction="horizontal" alignSelf="flex-start">
+        <Activity level={activityLevel} />
+      </Kb.Box2>
+      <Kb.Box2 direction="horizontal" gap="tiny" alignItems="center" style={styles.rightActionsContainer}>
+        {yourOperations.chat && <Kb.Button label="View" onClick={callbacks.onChat} small={true} />}
+        {yourOperations.editChannelDescription && (
+          <Kb.Button label="Edit" onClick={undefined /* TODO */} small={true} mode="Secondary" />
+        )}
+        <Kb.Button
+          label="Add members"
+          onClick={undefined /* TODO */}
+          ref={props.setAttachmentRef}
+          small={true}
+          mode="Secondary"
+          style={styles.addMembersButton}
+        />
+        <Kb.Button
+          mode="Secondary"
+          small={true}
+          ref={props.setAttachmentRef}
+          onClick={props.toggleShowingMenu}
+        >
+          <Kb.Icon type="iconfont-ellipsis" color={Styles.globalColors.blue} />
+        </Kb.Button>
+        {/* TODO: Channel Menu */}
+      </Kb.Box2>
+    </Kb.Box2>
+  )
+
+  if (Styles.isMobile) {
+    return (
+      <Kb.Box2 alignItems="flex-start" direction="vertical" fullWidth={true} style={styles.backButton}>
+        <Kb.Box2 direction="vertical" fullWidth={true} gap="small" style={styles.outerBoxMobile}>
+          <Kb.Box2 direction="horizontal" fullWidth={true} gap="tiny">
+            {topDescriptors}
+          </Kb.Box2>
+          {bottomDescriptorsAndButtons}
+          {yourOperations.manageMembers && <Kb.Box2 direction="horizontal" fullWidth={true}></Kb.Box2>}
+        </Kb.Box2>
+      </Kb.Box2>
+    )
+  }
+
+  return (
+    <Kb.Box2
+      alignItems="center"
+      direction="horizontal"
+      gap="small"
+      gapStart={true}
+      fullWidth={true}
+      className="headerTitle"
+    >
+      <Kb.Box2
+        direction="vertical"
+        alignItems="flex-start"
+        alignSelf="flex-start"
+        style={styles.outerBoxDesktop}
+      >
+        {topDescriptors}
+        {bottomDescriptorsAndButtons}
+      </Kb.Box2>
+    </Kb.Box2>
+  )
+}
+export default Kb.OverlayParentHOC(_HeaderTitle)
+
+const nyi = () => console.warn('not yet implemented')
+const useHeaderCallbacks = (_: TeamID, conversationIDKey: ConversationIDKey) => {
+  const dispatch = Container.useDispatch()
+
+  const onChat = () =>
+    dispatch(Chat2Gen.createPreviewConversation({conversationIDKey, reason: 'channelHeader'}))
```

Comment:
I tried this for a channel I wasn't in and got a kinda broken state:
![image](https://user-images.githubusercontent.com/4698687/74877463-79cae900-5333-11ea-9b4b-a97f43c8a8c7.png)

Reply:
Meaning you're an implicit admin? I'll test that.

Reply:
No - this is `keybase#android` and I'm not in that channel but I am in the team.

Reply:
Fixed now.